homeassistant/components/mqtt/client.py

```diff
@@ -654,9 +655,10 @@ def _matching_subscriptions(self, topic: str) -> list[Subscription]:
     @callback
     def _mqtt_handle_message(self, msg: mqtt.MQTTMessage) -> None:
         _LOGGER.debug(
-            "Received%s message on %s: %s",
+            "Received%s message on %s (qos=%s): %s",
```

Comment:
Although this is just logging I suggest to keep a similar format to the `Transmitting` debug log

Reply:
The difference here is that the payload is part of the log message.